src/Inventory/Home.js

```diff
@@ -35,14 +37,49 @@ export default class Home extends Component {
 						color={colors.tint}
 						size={24}
 						underlayColor='transparent'
-						onPress={() => navigation.navigate('CreateList')}
+						onPress={params.showNewListInput}
 						containerStyle={styles.rightButton}
 					/>
 				</View>
 			)
 		}
 	}
 
+	componentWillMount() {
+		this.props.navigation.setParams({ showNewListInput: this.showNewListInput });
```

Comment:
What is this for?

Reply:
It's to allow the header button to work with its screen components. I got it from the documentation [here](https://reactnavigation.org/docs/en/header-buttons.html#header-interaction-with-its-screen-component)